internal/testrunner/runners/system/test_config.go

```diff
@@ -24,7 +25,10 @@ import (
 	"github.com/elastic/elastic-package/internal/testrunner"
 )
 
-var systemTestConfigFilePattern = regexp.MustCompile(`^test-([a-z0-9_.-]+)-config.yml$`)
+var (
+	systemTestConfigFilePattern = regexp.MustCompile(`^test-([a-z0-9_.-]+)-config.yml$`)
+	allowedDeployerNames        = []string{"docker", "k8s", "tf"}
```

Comment:
Do we need to abbreviate? :slightly_smiling_face: And I think `compose` is more accurate than `docker`.
```suggestion
	allowedDeployerNames        = []string{"compose", "kubernetes", "terraform"}
```

Though I guess you are using the directory names we support?